### PR TITLE
Adds woodium toxin.

### PR DESCRIPTION
**Pull request by lordpidey**
## About The Pull Request

Adds a new toxin to the game, that doesn't directly hurt the victim, but rather subtly encourages the crew to murder them, by making them start reading WGW over the radio, all the while making them not realize they're reciting the world's best literature over the airwaves.

## Why It's Good For The Game

Giving traitors new and unique ways to take out targets is fun.

## Changelog
:cl:
add: Everyone's favorite literature now comes in liquid form, available wherever syndicate toxin kits are sold.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 1]
```diff
@@ -703,6 +703,51 @@
 		C.vomit(0, TRUE, TRUE, 4)
 		to_chat(C, "<span class='userdanger'>You feel something lumpy come up as you vomit.</span>")
 
+/datum/reagent/toxin/woodium
+	name = "Woodium"
+	id = "woodium"
+	description = "A toxin of unspeakable evil."
+	reagent_state = LIQUID
+	metabolization_rate = REAGENTS_METABOLISM
+	taste_description = "urine and flavored lube"
```
Comment (chesse20) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: yep this code looks good to me

**Review by chesse20 — APPROVED** [review]
code looks good to me

**Comment by PKPenguin321** [comment]
i kind of dont want this to be codified as an actual game mechanic, kind of ruins the joke in a way, you feel?

**Comment by wesoda25** [comment]
Naruto vs pain is honestly an incredible fight that i didnt give it credit for the first time i watched. The weird animation sort of threw me off, but now that I rewatch it perfectly conveyed the emotion of the scene. Pains repetition of the phrase “Do you hate me now” with varying levels of intensity show how he can even corrupt the main character who supposedly can do nothing morally wrong, bringing Naruto down to his level. This and his many other simple phrases throughout the battle add immense meaning to it, not wasting the emotion on long winded monologues. Some of my favorite parts include: narutos face right after pain cuts down hinata in front of him, when pain shouts “my pain is still far greater than yours”, everytime he says “do you hate me now”, and when he simply shouts “lets go” and charges at natuto. Its simple, but all the better because of it.

**Comment by HomocidlePancakes** [comment]
I rate this PR anal lube/69. Shouldn’t be made available to tots though because people will metaGAYm it and won’t lynch.

**Comment by optimumtact** [comment]
nope

**Pull request closed without merging**